pipelines/skylab/scATAC/ATAC.options.json

```diff
@@ -0,0 +1,6 @@
+{
+  "write_to_cache": true,
+  "read_from_cache": true,
+  "monitoring_script": "gs://broad-biccn-dev-test-data/monitoring.sh",
```

Comment:
Should this be updated to the gotc monitoring script that @jessicaway was talking about in slack this morning?